MSBuildTasks/FilterTestingConfigurations.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under 
+// the Apache License, Version 2.0 (the "License"); you may not use this 
+// file except in compliance with the License.  You may obtain a copy of the 
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT 
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the 
+// License for the specific language governing permissions and limitations
+// under the License.
+// 
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+
+namespace Remotion.BuildTools.MSBuildTasks
+{
+  public class FilterTestingConfigurations : Task
```

Comment:
This should be named `ValidateTestingConfigurations` (test class needs to be renamed too), as it does not filter anymore. 

@FlorianDecker Should the configurations get filtered too (would be a separate task now)? I guess this already happens somewhere else but I'm not sure. Will look into it.